lib/Screens/Home/Widgets/body.dart

```diff
@@ -8,6 +11,44 @@ class Body extends StatefulWidget {
 }
 
 class _BodyState extends State<Body> {
+  int questionnaireNotificationQuantity = 0;
+  int isSleepDiaryAnsweredToday = 0;
+  int readingNotificationQuantity = 0;
+
+  @override
+  void initState() {
+    super.initState();
+    getQuestionnaireNotification();
+  }
+
+  getQuestionnaireNotification() async {
+    await HelperFunctions.getUserEmailInSharedPreference().then((email) {
+      ScaleService()
+          .getQuestionnaireIsReadCount(email)
+          .then((notificationQuantity) {
+        setState(() {
+          questionnaireNotificationQuantity = notificationQuantity;
+        });
+      });
+      SleepService().isSleepDiaryAnsweredToday(email).then((answered) {
+        setState(() {
+          if (answered) {
+            isSleepDiaryAnsweredToday = 0;
```

Comment:
Esse atributo será somente para o SleepDiary?

Reply:
Se for so para o SleepDiary e representa se ja foi respondido, eh melhor trocar pra bool, nao?